src/app/features/posts/components/post-card/post-card.component.html

```diff
@@ -1,6 +1,16 @@
-<mat-card class="post-card">
-  <mat-card-title>
-    {{ title }}
-  </mat-card-title>
-  <img mat-card-image *ngIf="url" [src]="url" [routerLink]="['/posts', id]" />
-</mat-card>
+<div class="post-container">
+  <mat-card class="post-card">
+    <mat-card-title>
+      {{ title }}
+    </mat-card-title>
+    <img
+      mat-card-image
+      *ngIf="url"
+      [src]="url"
+      [routerLink]="['/posts', postId]"
+    />
+  </mat-card>
+
+  <msct-comments-preview [postId]="postId" class="preview-container">
+  </msct-comments-preview>
+</div>
```

Comment:
Moving
```html
<msct-comments-preview [postId]="postId" class="preview-container">
</msct-comments-preview>
```
Outside the component would increase post-card.components re-usability.

```html
   <post-page>
       (VIEW)
       <post-card>
           (VIEW)
           <comments-preview></comments-preview>
       </post-card>
   </post-page>
```
Is less reusable than:
```html
   <post-page>
       (VIEW)
       <post-card></post-card>
       <comments-preview></comments-preview>
   </post-page>
```